### ISSUE TITLE
"Funny item" balloon can be used to cancel targeting/spellcasting

### ISSUE THREAD

**Issue by hyddae** [ValveSoftware/Dota2-Gameplay#6442]
### Item name

Balloon

### Description

Example in Match ID: Ogre Magi casts Fireblast on Bounty Hunter. Bounty Hunter casts Balloon in between himself and Ogre Magi and Fireblast gets canceled.

### Example Match ID and Timestamp

6956649443 - 26:03

**Comment by shaileshgarg** [comment]
From the replay is looks like Ogre was the one who casted Ballon and not bounty.

**Comment by hyddae** [comment]
> From the replay is looks like Ogre was the one who casted Ballon and not bounty.

That is incorrect. I am the one playing Ogre. Bounty is the one putting up the balloon. I have witnessed this in other games also.

**Comment by shaileshgarg** [comment]
https://user-images.githubusercontent.com/26450759/211146021-7c13e0f2-5797-4310-8c82-a30620e92955.mp4

Ogre casted the Ballon not Bounty.
